packages/components/src/globals/scss/_theme-tokens.scss

```diff
@@ -87,6 +87,11 @@
   /// @group button
   $button-padding: calc(0.875rem - 3px) 60px calc(0.875rem - 3px) 12px !default !global;
 
+  /// @type Value
+  /// @access public
+  /// @group button
+  $button-padding-field: calc(0.675rem - 3px) 60px calc(0.675rem - 3px) 12px !default !global;
```

Comment:
Where do these numbers come from?

Reply:
@joshblack taken from the definition for `$button-padding-sm` beneath it